commands/new_function.go

```diff
@@ -136,3 +138,50 @@ func validTemplate(lang string) bool {
 
 	return found
 }
+
+func contains(s []string, e string) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
+
+func createORUpdateGitignore() {
+	FILES_TO_IGNORE := []string{"template", "build"}
+
+	f, err := os.OpenFile(".gitignore", os.O_RDWR|os.O_CREATE, 0644)
```

Comment:
does this need an `os.Stat` to check for the file before opening?

Reply:
It will create file if it is not present otherwise it will open it with read-write